cmake/CeleritasUtils.cmake

```diff
@@ -245,6 +255,20 @@ endfunction()
 
 #-----------------------------------------------------------------------------#
 
+macro(celeritas_setup_option var option) #[condition]
+  if(${ARGC} EQUAL 2)
+    # always-on-option
+    list(APPEND ${var}_OPTIONS ${option})
+  elseif(${ARGV2})
+    # variable evaluates to true
+    list(APPEND ${var}_OPTIONS ${option})
```

Comment:
Are those two branches supposed to have identical bodies?
Or is this a copy-and-paste oversight?

Reply:
They are supposed to be identical, and I *think* it's necessary because this is a cmake "macro" rather than a "function"...